lib/std/bounded_array.zig

```diff
@@ -25,12 +25,13 @@ pub fn BoundedArray(comptime T: type, comptime buffer_capacity: usize) type {
 /// Useful to pass around small explicitly-aligned arrays whose exact size is
 /// only known at runtime, but whose maximum size is known at comptime, without
 /// requiring an `Allocator`.
-/// ```zig
-//  var a = try BoundedArrayAligned(u8, 16, 2).init(0);
-//  try a.append(255);
-//  try a.append(255);
-//  const b = @ptrCast(*const [1]u16, a.constSlice().ptr);
-//  try testing.expectEqual(@as(u16, 65535), b[0]);
+///
+/// ```
+/// var a = try BoundedArrayAligned(u8, 16, 2).init(0);
```

Comment:
I know the code was like this before, but it might also be a good opportunity to fix the example to show that `BoundedArray` can be default-initialized with a 0 length without using `try`, e.g. `var a: BoundedArrayAligned(u8, 16, 2) = .{};`

Reply:
I agree! I never really use this `init`.